public/admin/config.yml

```diff
@@ -47,6 +47,66 @@ collections:
                   - { label: 'Alt text', name: 'altText', widget: 'string' }
                   - { label: 'Image', name: 'image', widget: 'image' }
                   - { label: 'Url', name: 'url', widget: 'string' }
+
+  - name: team
+    label: Team Directory
+    description: Team directory
+    files:
+      - name: team
+        label: Team Structure
+        file: 'src/cms-content/team/core-team.json'
+        fields:
+          - label: Teams
+            name: teams
+            widget: list
+            hint: 'Product Management, etc'
+            fields:
+              - label: Team Name
+                name: teamName
+                widget: string
+                hint: 'Product Management, Engineering, etc.'
+              - label: Team ID
+                name: teamId
+                widget: string
+                hint: 'ID for the URL (all lowercase, no spaces)'
```

Comment:
will we have URLs that link/scroll directly to certain teams?

Reply:
i guess either way IDs are always good to have

Reply:
there is a table of contents at the top (and maybe sidebar too), so we might need the IDs